src/glabs-inline/web/index.html

```diff
@@ -0,0 +1,15 @@
+<aside class="container">
+  <div class="thumbnail">
+    <a href="%%CLICK_URL_UNESC%%%%DEST_URL%%">
+      <img class="thumbnail__img" src="[%ThumbnailUrl%]">
```

Comment:
Can we add some alt text, or add `aria-hidden=true`? Just tried a screen reader and it reads out the src... 😅